serverless-functions/src/functions/features/park-interaction/flex/park-interaction.js

```diff
@@ -46,13 +51,43 @@ exports.handler = prepareFlexFunction(requiredParameters, async (context, event,
 
     if (webhookResult.success) {
       // Remove the agent
-      await InteractionsOperations.participantUpdate({
+      const removeAgentResponse = await InteractionsOperations.participantUpdate({
         context,
         interactionSid,
         channelSid,
         participantSid,
         status: 'closed',
       });
+      if (!removeAgentResponse.success) throw removeAgentResponse.message;
+
+      let createMapItem = {};
+      if (createUpdateSyncMapItem) {
+        // Open a Sync Map by unique name and update its data
+        const syncMap = await SyncOperations.createMap({
+          context,
+          uniqueName: workerName,
```

Comment:
I think using only worker name as the `uniqueName` could be problematic, because it could conflict with items created by other developers who might be short-sighted. We should add something to the name that indicates that it has something to do with parking. Also, I think we could also remove the need to pass workerName altogether if we use worker SID instead of name!

For example `ParkedInteractions_${workerSid}`

Reply:
Just made this change! Totally agree on having a more specified/hard to accidentally duplicate, unique name for the Map. This is one of those things you just pick something while developing and forget to change haha, another good catch, Sam!
And don't worry about requesting changes, I feel it gets more robust every time, and I'm learning a lot, so keep them coming if needed!